backend/apps/accounts/views.py

```
@@ -861,6 +861,70 @@ def get(self, request):
             status=status.HTTP_400_BAD_REQUEST,
         )
 
+class AvatarUploadView(APIView):
+    permission_classes = [IsAuthenticated]
+
+    def post(self, request):
+        serializer = AvatarUploadSerializer(data=request.data)
```

Ruff (0.15.20)

[error] Undefined name `AvatarUploadSerializer`

(F821)